internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1004,6 +1004,48 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"storage_blob_driver": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"storage_disk_driver": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
+						"version": {
+							Type:     pluginsdk.TypeString,
+							Optional: true,
+							Default:  "V1",
+							ValidateFunc: validation.StringInSlice([]string{
+								"V1",
+								"V2",
+							}, false),
+						},
+					},
+				},
+			},
+
+			"storage_file_driver": {
```

Comment:
Can we rename this to 
```suggestion
			"file_driver_enabled": {
```